packages/frontend/src/components/AccountListSidebar/AccountItem.tsx

```diff
@@ -99,7 +99,9 @@ export default function AccountItem({
         // set Timeout forces it to be run after react update
         setTimeout(() => {
           ActionEmitter.emitAction(KeybindAction.GlobalGallery_Open)
-        }, 0)
+          // NOTE(maxph): Gallery.tsx gets unmounted before receiving media data
+          // and breaks markdown, so here 50ms is a temprorary workaround for that
```

Comment:
What does "breaks markdown" mean?

Reply:
I don't have it on hand to reproduce but here's a screenshot from the video I posted above
![image](https://github.com/user-attachments/assets/fdec3b66-d10e-4c7d-b8ba-6f40cbc5058b)
It essentially inserts "All Media" in the chat header, effectively moving the chat title (with avatar to the middle of the header) and doesn't switch chat body to Gallery.

Reply:
so, it "breaks HTML" or "breaks Layout", it does not  "break markdown", which is not used here at all afaik 😅 

it is good to know what got broken, to be able to fix it 😅

Reply:
> so, it "breaks HTML" or "breaks Layout". it does not "break markdown", which is not used here at all afaik 😅
> 
> it is good to know what got broken, to be able to fix it 😅

Oh, sorry, tunnel vision. I meant markup, I thought HTML would be more confusing.